src/reducers/fileTree.tsx

```diff
@@ -43,19 +43,19 @@ const recursiveSortInPlace = (node: DirectoryNode): void => {
   });
 };
 
-const getVersionName = (version: Version) => {
-  return getLocalizedString(version.addon.name);
+const getAddonName = (versionAddonName: LocalizedStringMap) => {
+  return getLocalizedString(versionAddonName);
 };
 
-export const getRootPath = (version: Version) => {
-  return `root-${getVersionName(version)}`;
+export const getRootPath = (versionAddonName: LocalizedStringMap) => {
+  return `root-${getAddonName(versionAddonName)}`;
```

Comment:
So, that's probably a silly question but why do we need the name of the addon here? Couldn't we return `root` only? I am thinking about that because I am uncertain about the changes related to this function here.

Reply:
That's a valid question, which I hadn't even considered. The answer appears to be "no", which definitely simplifies things. Thanks for asking that, @willdurand!